mucgpt-frontend/src/components/StarterPrompt/StarterPrompt.module.css

```
@@ -23,73 +22,84 @@
-    box-shadow: none;
-    transition:
-        background-color 160ms ease,
-        border-color 160ms ease,
-        box-shadow 160ms ease,
-        transform 160ms ease;
+    padding: 8px 12px;
+    box-shadow: none !important;
     cursor: pointer;
 }
 
 .starterPrompt :global(.fui-Button__content) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)